chapter_13_database_layer_validation.asciidoc

```diff
@@ -1317,8 +1292,6 @@ And it's not designed for user-friendliness::
     Trying to store invalid data will cause a nasty `IntegrityError` to come
     back from your database, and possibly the user will see a confusing 500
     error page.
-    // DAVID: We haven't seen any IntegrityErrors here? This feels like it doesn't
-    // apply to this chapter.
```

Comment:
We have seen a few in earlier chapters but it's true, we have not triggered any in this one. But the statement is true though, attempting to store invalid data does result in IntegrityErrors. I would keep it.